server/modules/selva/module/selva_object.c

```diff
@@ -1336,7 +1336,35 @@ static void replyWithKeyValue(RedisModuleCtx *ctx, struct SelvaObjectKey *key) {
         break;
     case SELVA_OBJECT_OBJECT:
         if (key->value) {
-            replyWithObject(ctx, key->value);
+            if (key->user_meta == SELVA_OBJECT_META_SUBTYPE_TEXT) {
+                TO_STR(lang);
+                // fprintf(stderr, "YOYO %.*s\n", (int)lang_len, lang_str);
+                if (lang_len > 0) {
```

Comment:
if `lang == NULL` =>
```
  const char *RM_StringPtrLen(const RedisModuleString *str, size_t *len) {
      if (str == NULL) {
          const char *errmsg = "(NULL string reply referenced in module)";
          if (len) *len = strlen(errmsg);
          return errmsg;
      }
```

Reply:
Omg I forgot about this. Magnificent API...